selfdrive/car/mazda/carstate.py

```diff
@@ -0,0 +1,217 @@
+from cereal import car
+from selfdrive.config import Conversions as CV
+from opendbc.can.parser import CANParser
+from selfdrive.car.interfaces import CarStateBase
+from selfdrive.car.mazda.values import DBC, LKAS_LIMITS, CAR
+
+GearShifter = car.CarState.GearShifter
+
+class STEER_LKAS():
+  def __init__(self):
+    self.block = 1
+    self.track = 1
+    self.handsoff = 0
```

Comment:
doesn't seem like this is needed. block should be local to CS.update, tack is unused, and handsoff will be replaced by steerWarning